manifest.ttl

```diff
@@ -3,63 +3,98 @@
 @prefix p: <https://pipeline.described.at/> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 
-<filter> a p:Operation, p:Writable, p:Readable;
+<combine> a p:Operation, p:Writable, p:Readable;
+  rdfs:label "Combine";
+  rdfs:comment "Combines multiple streams to a single stream connecting them in the given order.";
+  code:implementedBy [ a code:EcmaScript;
+    code:link <node:barnard59-base#combine>
+  ].
+
+<concat> a p:Operation, p:Readable;
+  rdfs:label "Concat";
+  rdfs:comment "Concatenates the content of the given streams to a single stream.";
```

Comment:
I'm not sure I understand this comment, does it mean it's the same as `combine` except it doesn't care about the order?

Reply:
@vhf I can understand the confusion. I tried my best in the description, but don't claim it's the best description. Maybe it's easier to explain it based on 3 streams (`s1`, `s2`, `s3`) and how they are connected to the output stream (`o`). Is it more clear? How would you describe it?

### concat

Used if you have multiple inputs that should go to the same output.

```
s1 -> o
s2 -> o
s3 -> o
```

### combine

Used to create a sub-pipeline step. Should become deprecated once we are able to use a pipeline directly as a step.

```
s1 -> s2
s2 -> s3
s3 -> o
```

Reply:
Yes it's clear now, that was helpful, thanks!

IMO the combine description is fine (and it'll be deprecated anyway). For concat I'm not sure about the name, I don't see it as concatenating if it the order of the arguments don't matter. 
If this pseudocode represents what's happening here I'd call it merge (or join)?

```
o = merge(s1, s2, s3)
s2.push("a")
s3.push("b")
s3.push("c")
s1.push("d")
s3.push("e")
collect(o) == ["a", "b", "c", "d", "e"]
```

What I would expect concat to do, or what I understand when I read "concatenates …", would be 

```
o = concat(s1, s2, s3)
s2.push("a")
s3.push("b")
s3.push("c")
s1.push("d")
s3.push("e")
collect(o) == ["d", "a", "b", "c", "e"]
// because we basically did
// [o1, o2, o3] = [collect(s1), collect(s2), collect(s3)]
// o = [...o1, ...o2, ...o3]
```

Reply:
`concat` is actually doing what you describe that you would expect. My example ignored the time dimension. Like the operations `map` and `filter`, `concat` was aligned to the method names of `Array`.